apis/Google.Devtools.AspNet/Google.Devtools.AspNet/CloudErrorReporting/CloudErrorReportingExceptionLogger.cs

```diff
@@ -111,6 +133,12 @@ public override bool ShouldLog(ExceptionLoggerContext context)
             return context?.Exception != null;
         }
 
+        /// <returns>True if the client is not ready and too many tasks have already been created.</returns>
```

Comment:
Add remarks explaining that this has the side effect of updating the counter? (It sounds like it's a pure method at the moment.)